ddl/functions/calculate_artist_coin_fees.sql

```diff
@@ -0,0 +1,60 @@
+BEGIN;
+DROP FUNCTION IF EXISTS calculate_artist_coin_fees(TEXT);
+CREATE OR REPLACE FUNCTION calculate_artist_coin_fees(artist_coin_mint TEXT)
+RETURNS TABLE (
+    unclaimed_dbc_fees NUMERIC,
+    total_dbc_fees NUMERIC,
+    unclaimed_damm_v2_fees NUMERIC,
+    total_damm_v2_fees NUMERIC,
+    unclaimed_fees NUMERIC,
+    total_fees NUMERIC
+) LANGUAGE sql AS $function$
+    WITH
+    damm_fees AS (
+        -- fee = totalLiquidity * feePerTokenStore
+        -- precision: (totalLiquidity * feePerTokenStore) >> 128
+        -- See: https://github.com/MeteoraAg/damm-v2-sdk/blob/70d1af59689039a1dc700dee8f741db48024d02d/src/helpers/utils.ts#L190-L191
+        SELECT
+            pool.token_a_mint AS mint,
+            (
+                pool.fee_b_per_liquidity
+                * (
+                    position.unlocked_liquidity + position.vested_liquidity + position.permanent_locked_liquidity
+                )
+                / POWER (2, 128)
+                + position.fee_b_pending
+            ) AS total_damm_v2_fees,
```

Comment:
Are there docs somewhere where I can read how this stuff works? I'm just kind of guessing and I'm a little confused at what "total fees" means here.